src/components/Reports/GoalCalculator/Shared/GoalCalculatorContext.tsx

```diff
@@ -62,14 +70,19 @@ export const GoalCalculatorProvider: React.FC<Props> = ({ children }) => {
   const [selectedStepId, setSelectedStepId] = useState(
     GoalCalculatorStepEnum.CalculatorSettings,
   );
+  const [selectedReport, setSelectedReport] =
+    useState<GoalCalculatorReportEnum>(GoalCalculatorReportEnum.MpdGoal);
   const [rightPanelContent, setRightPanelContent] =
     useState<JSX.Element | null>(null);
   const [isDrawerOpen, setIsDrawerOpen] = useState<boolean>(true);
 
-  const currentStep = useMemo(
-    () => steps.find((step) => step.id === selectedStepId),
-    [steps, selectedStepId],
-  );
+  const currentStep = useMemo(() => {
+    const currentStep = steps.find((step) => step.id === selectedStepId);
```

Comment:
Probably best to call `currentStep` something else for readability since the function name is `currentStep`